editors/sc-ide/CMakeLists.txt

```diff
@@ -350,21 +350,29 @@ if(APPLE)
                 #fixup_qt5_executable(\"${CMAKE_INSTALL_PREFIX}/SuperCollider/SuperCollider.app\")
                 " COMPONENT Runtime)
 
-
+elseif(CMAKE_GENERATOR MATCHES "Visual Studio")
+  set_property(TARGET SuperCollider PROPERTY RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/editors/sc-ide")
+  install(TARGETS SuperCollider
+      CONFIGURATIONS Debug;Release
+      DESTINATION "SuperCollider"
+      PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
+  install(FILES ${translations}
+      CONFIGURATIONS Debug;Release
```

Comment:
The problem here is the following: with the multi configuration VS generator the "build configuration" or "cmake build type" is not determined in the the cmake configure step, but as an argument to "make". "Make" expects alternative configurations to be available for the build. In the case of portaudio, you get errors if you use a static library compiled as release version in a SC debug build and vice versa. So rather than just switch the configuration, you'd have to point to different port-audio source as well, which is a bit hassle. This change allows to access differently configured portaudio libraries. I know there are more configurations, but - at least at the time I did this - cmake only allowed to handle two configurations in this manner. Do you know an alternative way to handle the underlying problem?
On a side line: Yesterday @sonoro1234 published a portaudio version that builds successfully with cmake using the MinGW compiler, and can also be used to compile with VS (https://github.com/sonoro1234/portaudio20111121) - the problem I tried to solve with above changes would go away if portaudio was added to sc as external module. Then portaudio will just be compiled in the configuration chosen for the SC build. I know you actually did that in the past, but then removed it again. What is your take on adding portaudio to the SC sources? Why did you remove it in the past?